sdk/messaging/azwebpubsub/example_test.go

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azwebpubsub_test
+
+import (
+	"log"
+	"os"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+	"github.com/Azure/azure-sdk-for-go/sdk/messaging/azwebpubsub"
+)
+
+func ExampleNewClientWithDefaultAzureCredential() {
```

Comment:
Per the `go vet` failure I believe you need a `_` between `Example` and the suffix (same comment applies to the other example).